### ISSUE TITLE
CD based installers don't work

### ISSUE THREAD

**Issue by juligreen** [flathub/net.lutris.Lutris#183]
**Describe the problem**
As the flatpak does not have access to /run/media (this was the default cd path on archlinux with cdemu) you currently can't use CD based installers.

By using flatseal I was able to manually give access to "/run/media:ro". This enabled me to manually specify the cd's path, but autodetection still did not work

**Is it reproducible with native Lutris?**
no, native Lutris is able to automatically detect files on cds and use them

**Expected behavior**
The flatpak should behave the same as native Lutris

**Flatpak info**
`flatpak --version` - Flatpak 1.11.3

`flatpak --gl-drivers`:
nvidia-470-74
default
host

**Comment by juligreen** [comment]
While I haven't replicated my exact use-case from before, I believe this can be fixed with `  - --system-talk-name=org.freedesktop.UDisks2`

**Comment by strycore** [comment]
Note that a game installer could be located anywhere on the user's system, not just in subdirectories of /run/media

**Comment by MinmoTech** [comment]
That's true, this issue was only for CD based installers which should be fixed by https://github.com/flathub/net.lutris.Lutris/pull/226

(I haven't tested it yet though)
